src/components/marker-table/index.css

```diff
@@ -11,30 +11,27 @@
 }
 
 .treeViewFixedColumn.start {
-  left: 0;
   width: 80px;
   padding-right: 5px;
   text-align: right;
 }
 
 .treeViewFixedColumn.duration {
-  left: 80px;
   width: 80px;
   padding-right: 5px;
   border-right: none;
   text-align: right;
 }
 
+.treeViewHeaderColumn.name {
+  padding-left: 20px;
```

Comment:
The `left` value was taking this into account previously. This padding accounts for some padding happening in the tree itself as a result of not having the expanding button. Maybe we should look at removing the expanding button for flat "trees" later.